src/model_repository_manager.cc

```diff
@@ -546,11 +554,14 @@ class ModelRepositoryManager::ModelLifeCycle {
   ModelLifeCycle(
       const double min_compute_capability, InferenceServer* server,
       const triton::common::BackendCmdlineConfigMap& backend_cmdline_config_map,
-      const triton::common::HostPolicyCmdlineConfigMap& host_policy_map)
+      const triton::common::HostPolicyCmdlineConfigMap& host_policy_map,
+      const unsigned int model_load_thread_count)
```

Comment:
It may make sense to use a `option` struct to group the configurations. Can be done in separate change / ticket

Reply:
Added DLIS-3866